lib/Styliner.js

```diff
@@ -126,6 +126,27 @@ function parseDataUri(uri) {
 
 
 /**
+ * Asynchronously parses an HTML document from file path passed as argument and inlines styles as appropriate.
+ *
+ * @param {String}			filePath				The file path to parse HTML document
+ */
+Styliner.prototype.parseFile = function(filePath) {
+	var _this = this;
+	var defer = Q.defer();
+  fs.readFile(filePath, 'utf8', function(err, content) {
```

Comment:
Please fix indentation.
Also, please use `qfs.read()` to read files instead of mucking around with Deferreds.